### ISSUE TITLE
Summon Screen

### ISSUE THREAD

**Issue by Kaytila** [slashware/NovaMundiTracker#49]
Barter/Trade and so on have a "ok" Button, "Summon" Screen does not.
So if I pick the wrong person by accident, I have to stick with him.
I would like to see a ok/reset button set at the same place as on Barter screen here.

**Comment by slashman** [comment]
I agree UX should be unified. However please note that you can click on the person you added by mistake, to transfer him back for free into the town and get back your experienceS points.

**Comment by Kaytila** [comment]
You can? but how do i recognize the person? :D

**Comment by slashman** [comment]
It doesn't say "party member" :P

**Comment by Kaytila** [comment]
see? never saw that ...
